src/elektra/redshift.ini

```diff
@@ -0,0 +1,308 @@
+type = string
+check/date = ISO8601
+check/date/format = timeofday
+description = Instead of using the solar elevation at the user's location, the time intervals of dawn and dusk can be specified manually (source: https://github.com/jonls/redshift/wiki/Configuration-file#custom-dawndusk-intervals).
+default = 19:00
+example = 20:00
+opt/long = time-dusk-start
+opt/arg = required
+
+[provider/time/dusk/end]
+type = string
+check/date = ISO8601
+check/date/format = timeofday
+description = Instead of using the solar elevation at the user's location, the time intervals of dawn and dusk can be specified manually (source: https://github.com/jonls/redshift/wiki/Configuration-file#custom-dawndusk-intervals).
+default = 20:30
+example = 21:30
+opt/long = time-dusk-end
+opt/arg = required
+
+[provider/location]
+type = enum
+check/enum = #4
+check/enum/#0 = corelocation
+check/enum/#1 = geoclue2
+check/enum/#2 = manual
+check/enum/#3 = auto
+check/enum/#4 = list
+description = The location provider to be used. By default, one of the supported location providers on the current OS will be chosen automatically. The provider is used to establish whether it is currently daytime or nighttime.
+default = auto
+example = geoclue2
+opt/long = location-provider
+opt/arg = required
+
+[provider/location/help]
+type = boolean
+description = Prints the help of the location providers.
+default = 0
+example = 1
+opt/long = help-providers
+opt/arg = none
+
+[provider/location/elevation/high]
+type = float
+description = By default, Redshift will use the current elevation of the sun to determine whether it is daytime, night or in transition (dawn/dusk). When the sun is above the degrees specified with the elevation-high key it is considered daytime and below the elevation-low key it is considered night (source: https://github.com/jonls/redshift/wiki/Configuration-file#solar-elevation-thresholds). Affects all location providers.
+default = 3.0
+example = 3.5
+opt/long = elevation-high
+opt/arg = required
+
+[provider/location/elevation/low]
+type = float
+description = By default, Redshift will use the current elevation of the sun to determine whether it is daytime, night or in transition (dawn/dusk). When the sun is above the degrees specified with the elevation-high key it is considered daytime and below the elevation-low key it is considered night (source: https://github.com/jonls/redshift/wiki/Configuration-file#solar-elevation-thresholds). Affects all location providers.
+default = -6.0
+example = -5.0
+opt/long = elevation-low
+opt/arg = required
+
+[provider/location/manual/lat]
+type = float
+description = The location latitude. Only applies to location provider "manual".
+check/type = float
+check/range = -90.0-90.0
+; Latitude of berlin:
+default = 52.520008
+example = 52.520008
+opt/long = lat
+opt/arg = required
+
+[provider/location/manual/lon]
+type = float
+description = The location longitude. Only applies to location provider "manual".
```

Comment:
Write that negative coordinates are needed for America.

Reply:
Thanks, I improved the description for `lat` and `lon`.